### ISSUE TITLE
(feature request) Password to secure app

### ISSUE THREAD

**Issue by zeoint** [twofas/2fas-android#47]
Currently the app uses 4-digit, 6-digit and biometrics to lock.
It would be nice to have the ability to be able to set a password to secure the app.

**Comment by 2fas-com** [comment]
Thanks for the suggestion. We're already planning to add this, but we can't tell the exact date as there are other priorities right now.

**Comment by Tipoff4317** [comment]
Yes, please.  Allow long password to encrypt to the data, and allow biometrics / limited PIN attempts to access the data, falling back to password if there are too many attempts.  This would be ideal.  Thx!
